model.py

```diff
@@ -437,6 +438,48 @@ def in_production(self):
         prod = self.PRODUCTION_STAGE
         return self.library_stage == prod and self.registry_stage == prod
 
+    @property
+    def service_area_name(self):
+        """Describe the library's service area in a short string a human would
+        understand, e.g. "Kern County, CA".
+
+        This library does the best it can to express a library's service
+        area as the name of a single place, but it's not always possible
+        since libraries can have multiple service areas.
+
+        TODO: We'll want to fetch a library's ServiceAreas (and their
+        Places) as part of the query that fetches libraries, so that
+        this doesn't result in extra DB queries per library.
+
+        :return: A string, or None if the library's service area can't be
+           described as a short string.
+        """
+        # Group the ServiceAreas by type.
+        by_type = defaultdict(set)
+        for a in self.service_areas:
+            if not a.place:
+                continue
+            if a.place.type == Place.EVERYWHERE:
+                # We already know that 'everywhere' won't work. Ignore
+                # it so it doesn't mask something more specific.
+                continue
+            by_type[a.type].add(a)
+
+        # If there is a single focus area, use it.
+        # Otherwise, if there is a single eligibility area, use that.
```

Comment:
I'll preface this by saying I don't know the data enough to actually know this is useful, so feel free to ignore. I wonder though if there's a potential low-lift win for cases where a library's service area is a list of zipcodes, where the service area name is expressed as `f"ZIP codes {', '.join(list_of_zips)}"`?

Reply:
Possibly even for any library where there are multiple service areas but they're all the same type, like `f"{service_area_type_plural}: {', '.join(service_area_names)}"`

Reply:
It's possible to generate a string like this, but the mockups don't show enough space for it, and I don't think seeing something that complicated would help anyone. I think the 'right' answer here is to make it possible for the library admin to set this to a freeform value like 'SF Bay Area'.